### ISSUE TITLE
YouTube TV Support

### ISSUE THREAD

**Issue by bjsafdie** [GoogleChromeLabs/picture-in-picture-chrome-extension#29]
It would be fantastic if this worked with YouTube TV.

**Comment by nojvek** [comment]
It would be fantastic, if Youtube's picture in picture button actually did `video.requestPictureInPicture` rather than do a half assed p-in-p that only works on the youtube page . 🤦‍♂️

**Comment by beaufortfrancois** [comment]
YouTube TV disabled Picture-in-Picture with a [feature policy](https://w3c.github.io/picture-in-picture/#feature-policy), which prevents Chrome to request Picture-in-Picture for any video in tv.youtube.com.

**Comment by polywock** [comment]
Exactly, but there are workarounds. I made an extension that unblocks PiP functionality by removing the PiP blocking Feature-Policy headers. If you're concerned about the permission level, you could go to tv.youtube.com and right click extension to limit the Site Access to only Youtube TV. 

https://github.com/polywock/pipUnblocker
